tests/integration/parameters.test.ts

```diff
@@ -59,6 +57,25 @@ describe('queryParametersValidation', () => {
     expect(resp.errors.p_string).toBeUndefined()
   })
 
+  test('checkStringDecodeValid', async () => {
+    const encoded1 = '&p_str=+%20+'
+    const encoded2 = encodeURIComponent('+ +') // %2B%20%2B
+    const qs =
+      '?p_enumeration=csv&p_enumeration_insensitive=CSV&p_string=' +
+      encoded1 +
+      encoded2
+    const request = await todoRouter.handle(
+      buildRequest({ method: 'GET', path: `/todos${qs}` })
+    )
+    const resp = await request.json()
+    expect(request.status).toEqual(200)
+
+    expect(resp.results).toEqual(['lorem', 'ipsum'])
+    // FIXME: the query params not properly returned here
```

Comment:
I tried to trace the response, but I couldn't figure it out...need some help here. thx